storybook/src/components/SearchField/SearchField.stories.tsx

```diff
@@ -75,7 +75,6 @@ export const Controlled: any = {
           const value = formData.get('search-box')
           // search actions should not be triggered without a value
           if (value) {
-            // eslint-disable-next-line no-alert
```

Comment:
I think this should trigger an ESLint warning? We don't want to use `alert` in our code, I think?

Reply:
Restored.